web/packages/teleport/src/UnifiedResources/Resources.tsx

```diff
@@ -124,34 +110,58 @@ export function Resources() {
         pathname={pathname}
         replaceHistory={replaceHistory}
       />
-      {attempt.status === 'failed' && (
-        <ErrorMessage message={attempt.statusText} />
-      )}
       <ResourcesContainer gap={2}>
-        {fetchedData.agents.map((agent, i) => (
-          <ResourceCard key={i} onLabelClick={onLabelClick} resource={agent} />
+        {resources.map((res, i) => (
+          <ResourceCard key={i} onLabelClick={onLabelClick} resource={res} />
         ))}
       </ResourcesContainer>
-      <div
-        ref={infiniteScrollDetector}
-        style={{
-          visibility: attempt.status === 'processing' ? 'visible' : 'hidden',
-        }}
-      >
-        {(attempt.status === 'processing' || fetchedData.startKey) && (
-          <Box
-            textAlign="center"
-            style={{ visible: attempt.status === 'processing' }}
-          >
-            <Indicator />
-          </Box>
+      <div ref={scrollDetector} />
+      <ListFooter>
+        <IndicatorContainer status={attempt.status}>
+          <Indicator size={`${indicatorSize}px`} />
+        </IndicatorContainer>
+        {attempt.status === 'failed' && resources.length > 0 && (
+          <ButtonSecondary onClick={retryClicked}>Load more</ButtonSecondary>
         )}
-      </div>
+      </ListFooter>
     </FeatureBox>
   );
 }
 
+const indicatorSize = 48; //px
```

Comment:
```suggestion
const INDICATOR_SIZE = 48; //px
```

it looks like we aren't using this number is any calculations anywhere and just appending `px` to it when used. should this just be a string = `48px` instead of the suggestion?